### ISSUE TITLE
[Suggestion]: 

### ISSUE THREAD

**Issue by Reaperjm** [DexxKnight1/AOF-The-Frozen-Hope#1]
### General Info

- [ ] I would like to suggest a new mod
- [ ] I would like to suggest a new quests
- [X] I would like to propose changes
- [ ] I would like to propose another thing

### Suggestion Description

Missing recipes (can't code myself so not sure how to add it) for at least lead ingots to blocks via crafting table. This makes compact drawers not work for at least that ingot. Unless it was intended for lead to only be made into blocks via thermal compressor device.

**Comment by Reaperjm** [comment]
So I went through all the ingots. Any ingot not specifically with a recipe from another mod (greek fantasy for example with bronze) unified by the unifer mod do not have a ingot to block recipe. Lead, silver, tin, nickel are just naming ones I confirmed lack this recipe. All vanilla ore work just fine. Plus ingots added buy other mods such as bronze greek fantasy, ATM ingots, and tinkers stuff. Not sure why those ones don't have anything, but they don't. I am not sure how to add them myself otherwise I would. Not even sure where to put it as an issue. Here it is though, so sure recipe updates coming soon.
